### PR TITLE
Update possible k8s manifest names

### LINKED ISSUE #2857: update possible names a k8s manifest can have
we should add **manifest, manifest.yml, manifest.yaml** as possible names of a k8s manifest.

### PR DESCRIPTION
**Pull request by jeremyphua**
Signed-off-by: Jeremy Phua <jeremyphuachengtoon@gmail.com>

Fixes #2857

## Proposed changes

- Add **manifest**, **manifest.yml**, **manifest.yaml** to manifestSubPath slice in `pkg/model/manifest.go`.

### CODE REVIEW

File: pkg/model/manifest.go [hunk 1]
```diff
@@ -124,6 +124,9 @@ var (
 		"helm/charts",
 	}
 	manifestSubPath = []string{
+		"manifest",
+		"manifest.yml",
+		"manifest.yaml",
```
Comment (ifbyol) [inline pkg/model/manifest.go:129]: As the files are processed in the order defined here and in order to keep current preference of files, can you include them at the end of the list?

**Review by ifbyol — COMMENTED** [review]
Thanks for your contribution!

File: pkg/model/manifest.go (hunk 1 above)
Comment (jeremyphua) [reply pkg/model/manifest.go]: Sure, I have made the changes!

**Review by jLopezbarb — APPROVED** [review]

**Review by teresaromero — APPROVED** [review]

File: pkg/model/manifest.go (hunk 1 above)
Comment (jeremyphua) [reply pkg/model/manifest.go]: Thanks for reviewing @ifbyol!

**Pull request merged**